test/Conversion/TorchToLinalg/basic.mlir

```diff
@@ -95,6 +95,44 @@ func.func @torch.aten.Int.Tensor$non_zero_rank(%arg0: !torch.vtensor<[?,?],si64>
 
 // -----
 
+// CHECK-LABEL:     func.func @torch.aten.Int.Tensor$zero_rank
+// CHECK-SAME:          (%[[ARG:.*]]: !torch.vtensor<[],ui8>) -> !torch.int {
+// CHECK:               %[[I:.*]] = torch_c.to_builtin_tensor %[[ARG]] : !torch.vtensor<[],ui8> -> tensor<ui8>
+// CHECK:               %[[EXT:.*]] = tensor.extract %[[I]][] : tensor<ui8>
+// CHECK:               %[[RET:.*]] = torch_c.from_ui8 %[[EXT]]
+// CHECK:               return %[[RET]] : !torch.int
+func.func @torch.aten.Int.Tensor$zero_rank(%arg0: !torch.vtensor<[],ui8>) -> !torch.int {
+  %0 = torch.aten.Int.Tensor %arg0 : !torch.vtensor<[],ui8> -> !torch.int
+  return %0 : !torch.int
+}
+
+// -----
+
+// CHECK-LABEL:     func.func @torch.aten.Int.Tensor$non_zero_rank
+// CHECK-SAME:          (%[[ARG:.*]]: !torch.vtensor<[?,?],si64>) -> !torch.int {
```

Comment:
I think the input dtype here should be `ui8`

Reply:
Got it fixed. But still got similar bug:

```
Input was:
<<<<<<
           42: } 
           43:  
           44:  
           45: // ----- 
           46: module { 
           47:  func.func @torch.aten.Int.Tensor$non_zero_rank(%arg0: !torch.vtensor<[?,?],si64>) -> !torch.int { 
label:98'0                                                    X~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ error: no match found
           48:  %0 = torch_c.to_builtin_tensor %arg0 : !torch.vtensor<[?,?],si64> -> tensor<?x?xi64> 
label:98'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           49:  %c0 = arith.constant 0 : index 
label:98'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           50:  %1 = tensor.dim %0, %c0 : tensor<?x?xi64> 
label:98'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           51:  %c1 = arith.constant 1 : index 
label:98'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           52:  %2 = tensor.dim %0, %c1 : tensor<?x?xi64> 
label:98'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          105: } 
label:98'0     ~~
          106:  
label:98'0     ~
          107:  
label:98'0     ~
          108: // ----- 
label:98'0     ~~~~~~~~~
          109: module { 
label:98'0     ~~~~~~~~~
          110:  func.func @torch.aten.Bool.Tensor$zero_rank(%arg0: !torch.vtensor<[],i1>) -> !torch.bool { 
label:98'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
label:98'1      ?                                                                                           possible intended match
          111:  %0 = torch_c.to_builtin_tensor %arg0 : !torch.vtensor<[],i1> -> tensor<i1> 
label:98'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          112:  %c1_i64 = arith.constant 1 : i64 
label:98'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          113:  %c0 = arith.constant 0 : index 
label:98'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          114:  %1 = tensor.extract %0[] : tensor<i1> 
label:98'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          115:  %2 = torch_c.from_i1 %1 
label:98'0     ~~~~~~~~~~~~~~~~~~~~~~~~~
>>>>>>

```

Reply:
Ah, I think the issue is that the tests have the same name as the tests for `si64`. Can you give them a unique name, such as `func.func @torch.aten.Int.Tensor$non_zero_rank_bool` and check if that fixes it?